app/javascript/oldjs/miq_flash.js

```diff
@@ -60,7 +60,18 @@ window.add_flash = function(msg, level, options) {
     textDiv.attr('id', options.id);
   }
 
-  $('#flash_msg_div').append(textDiv).show();
+  var listOfFlashDivs = document.getElementsByClassName('flash_text_div')
+  var foundFlashMessage = false;
+  listOfFlashDivs.forEach((oneDiv) => {
+    var innerText = oneDiv.innerText.replace(/(\r\n|\n|\r)/gm, "")
+    if (innerText === textDiv[0].innerText) {
+      foundFlashMessage = true;
+      break;
+    }
+  })
+  if (!foundFlashMessage) {
+    $('#flash_msg_div').append(textDiv).show();
+  }
```

Comment:
Can you explain what this is doing?  Offhand it looks like it's just seeing if there are more than one flash message and then picking the first one.  If so, that's gonna cause problems where we actually want multiple flash messages (usually for different, but multiple errors, or sometimes a warning + and error).  One example place is on the advanced settings page, I believe (I'll try and get a screenshot)